src/config/dark-sites.config

```diff
@@ -973,6 +974,7 @@ wowaudit.com
 wowhead.com
 wowinterface.com
 wtfast.com
+www.canalplus.com
```

Comment:
I don't know about other languages, in Polish the main page seems to be dark, but help and other pages are light

Reply:
@nikkehtine  see

https://kup.pl.canalplus.com/oferta/wybierz-pakiet

Rule should be more general (without `www.` prefix)

CC @Gusted 

![Uploading 20220106-1641497520.png…]()